app/src/main/java/com/example/arc_exapmle/note/NoteRepository.kt

```diff
@@ -5,132 +5,68 @@ import android.os.AsyncTask
 import androidx.lifecycle.LiveData
 import com.example.arc_exapmle.StarDatabase
 import com.example.arc_exapmle.user.UserEntity
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers.Default
+import kotlinx.coroutines.launch
 
-class NoteRepository(application: Application , val userId : Int) {
-    private var allNotes: LiveData<List<NoteEntity>>
-    private var noteDao: NoteDao
+class NoteRepository(dataBase: StarDatabase, private val userIdNumber: Int) {
```

Comment:
age faghat be `NoteDAO` niaze inja chera az hamon aval faghat noteDao ro nagirim az biron?

Reply:
dorost migin hallesh kardam